sktree/tests/test_supervised_forest.py

```diff
@@ -469,20 +469,20 @@ def test_check_importances_oblique_regressor(criterion, dtype, feature_combinati
         n_estimators=10,
         criterion=criterion,
         random_state=0,
-        feature_combinations=feature_combinations,
+        max_bins=3,
```

Comment:
I would not touch max_bins as that is equivalent to making al the data into 3. I would also not touch the feature_combinations as that messes the existing test.

There is another test that you can add the regressor test into: `test_check_importances_oblique`.